Resources/config/core.xml

```diff
@@ -66,7 +66,7 @@
         </service>
         <!-- twig -->
         <service id="sonata.admin.twig.global" class="Sonata\AdminBundle\Twig\GlobalVariables">
-            <argument type="service" id="service_container"/>
+            <argument type="service" id="sonata.admin.pool"/>
```

Comment:
If they inherit from this service definition, yes, there is a BC-break (this service should probably be private to avoid this kind of problem).